### ISSUE TITLE
Dynamic identifier on input from $scope value

### ISSUE THREAD

**Issue by vilmysj** [danialfarid/ng-file-upload#1529]
I need dynamic id's or other identifier on label/input elements for Selenium tests.
It seems that current implementation do not compile/evaluate values from scope.

For example
```js
<div id="myid-{{scopeValue}}" ngf-select></div>
```
gives
```js
<input type="file"  id="ngf-myid-{{scopeValue}}" ...>
```

here is jsfiddle: http://jsfiddle.net/sg5d1cuq/

In my case I have file tree where each folder uses `ng-file-upload` and handles upload.
File tree is generated using `ng-repeat` so id's can't be hard-coded.

**Comment by vilmysj** [comment]
Maybe someone know workaround for this?

**Comment by lucasBertola** [comment]
+1

**Comment by wenchaojiang** [comment]
+ 1

**Comment by balajicbabu** [comment]
+1

**Comment by ashwinaggarwal** [comment]
+1

**Comment by danialfarid** [comment]
Fixed at 12.2.12
